timed/settings.py

```diff
@@ -11,7 +12,7 @@
 django_root = environ.Path(__file__) - 2
 
 ENV_FILE = env.str("DJANGO_ENV_FILE", default=django_root(".env"))
-if os.path.exists(ENV_FILE):  # pragma: no cover
+if Path(ENV_FILE):  # pragma: no cover
```

Comment:
I think this is an antipattern. At least call `.exists()` on it, so your intention is clear when reading the code

```suggestion
if Path(ENV_FILE).exists():  # pragma: no cover
```

Reply:
i actually forgot the .exists there

Reply:
And I thought you knew something I didn't - that `bool(Path("non-existing-file"))` might magically imply an existance check. Glad I caught this, because it doesn't. `bool(Path("non-existing")) is True`

Reply:
wait, do we need to learn how write new ruff rules for this :grin: